Gruntfile.js

```diff
@@ -47,17 +47,13 @@ module.exports = function(grunt) {
     jscs: {
       src: 'samples/web/content/**/*.js',
       options: {
-        preset: 'google', // as per Google style guide – could use '.jscsrc' instead
+        config: 'samples/web/.jscsrc', // uses Google preset with some exceptions
```

Comment:
I think overall we should aim at the google preset with no modifications/exceptions.

Is fixing the 100chars lines and the multipleVarDecl too much to go in this CL?
Or you plan to never fix them?
If the first, then add a TODO and follow up with a CL to clean it up.